content/docs/gherkin/step-organization.md

```diff
@@ -124,6 +126,6 @@ This helps in a number of ways,
 
 You can handle other behaviours, like *validating a webpage, clicking a button, etc.*, the same way.
 
-We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.
+{{% text "java" %}}We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.{{% /text %}}{{% text "kotlin" %}}We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.{{% /text %}}
```

Comment:
```suggestion
{{% text "java,kotlin" %}}We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.{{% /text %}}
```

Reply:
Please use polyglot annotation for Java and Kotlin together for easier maintenance.

Reply:
I’ll be happy to! I didn’t know that syntax existed.